resource/source-python/translations/wcs/menu_strings.ini

```diff
@@ -1,409 +1,593 @@
     en = "Unused points: {unused}"
+	ru = "Неиспользуемые очки: {unused}"
 [spendskills_menu skill]
     en = "{name} ({level}/{maxlevel})"
+	ru = "{name} ({level}/{maxlevel})"
 [spendskills_menu required]
     en = "{name} (need level {required})"
+	ru = "{name} (треб. уровень {required})"
 [spendskills_menu max]
     en = "{name} (MAX)"
+	ru = "{name} (макс.)"
 
 [changerace_menu title]
     en = "Changerace Menu"
+	ru = "[ВЫБОР РАСЫ]"
 [changerace_menu line]
     en = "{name} (level: {level})"
+	ru = "{name} (уровень: {level})"
 [changerace_menu maximum level]
     en = "{name} (maximum level: {level})"
+	ru = "{name} (макс. уровень: {level})"
 [changerace_menu required level]
     en = "{name} (required level: {level})"
+	ru = "{name} (треб. уровень: {level})"
 [changerace_menu team limit]
     en = "{name} (team limit: {count})"
+	ru = "{name} (лимит на команду: {count})"
 [changerace_menu team]
     en = "{name} (restricted team: {team})"
+	ru = "{name} (запр. команда: {team})"
 [changerace_menu private]
     en = "{name} (private)"
+	ru = "{name} (приватная)"
 [changerace_menu map]
     en = "{name} (restricted map: {map})"
+	ru = "{name} (запр. карта: {map})"
 
 [raceinfo_menu title]
     en = "Raceinfo Menu"
+	ru = "[ИНФО О НАВЫКАХ]"
 
 [raceinfo_detail_menu title]
     en = "Raceinfo - {name} ({place}/{total})"
+	ru = "[ИНФО О {name} ({place}/{total})]"
 [raceinfo_detail_menu required]
     en = "Required level: {required}"
+	ru = "Треб. уровень: {required}"
 [raceinfo_detail_menu maximum]
     en = "Maximum level: {maximum}"
+	ru = "Макс. уровень: {maximum}"
 [raceinfo_detail_menu author]
     en = "Author: {name}"
+	ru = "Автор: {name}"
 [raceinfo_detail_menu public]
     en = "<Public race>"
+	ru = "<Общая раса>"
 [raceinfo_detail_menu private]
     en = "<Private race>"
+	ru = "<Приватная раса>"
 [raceinfo_detail_menu private allowed]
     en = "<Private race (allowed)>"
+	ru = "<Приватная раса (разрешено)>"
 [raceinfo_detail_menu skills]
     en = "Skills: {count}"
+	ru = "Навыков: {count}"
 [raceinfo_detail_menu description]
     en = "Description"
+	ru = "Описание"
 [raceinfo_detail_menu back]
     en = "Previous race"
+	ru = "Пред. раса"
 [raceinfo_detail_menu next]
     en = "Next race"
+	ru = "След. раса"
 
 [raceinfo_skills_menu title]
     en = "Raceinfo - {name}"
+	ru = "[ИНФО О {name}]"
 [raceinfo_skills_menu ultimate]
     en = "{name} (ultimate)"
+	ru = "{name} (ульт)"
 [raceinfo_skills_menu ability]
     en = "{name} (ability)"
+	ru = "{name} (способность)"
 [raceinfo_skills_menu ability_on]
     en = "{name} (+ability {index})"
+	ru = "{name} (+ability {index})"
 
 [raceinfo_skills_detail_menu title]
     en = "Raceinfo - {name}"
+	ru = "[ИНФО О {name}]"
 [raceinfo_skills_detail_menu description]
     en = "Skill: {name}"
+	ru = "Навык: {name}"
 
 [raceinfo_race_detail_menu title]
     en = "Raceinfo - {name}"
+	ru = "[ИНФО О {name}]"
 
 [playerinfo_menu title]
     en = "Playerinfo Menu"
+	ru = "Playerinfo Menu"
```

Comment:
Did you forget to translate these 3 messages? "playerinfo_menu online" and "playerinfo_menu offline".